pkg/cmd/pipeline/start.go

```diff
@@ -394,6 +401,19 @@ func mergeRes(pr *v1alpha1.PipelineRun, optRes []string) error {
 	return nil
 }
 
+func mergeLabels(pr *v1alpha1.PipelineRun, labelPar []string) error {
+	labels, err := parseLabels(labelPar)
+	if err != nil {
+		return err
+	}
+	if len(labels) == 0 {
+		return nil
+	}
+
+	pr.ObjectMeta.Labels = labels
```

Comment:
Just a thought, this will remove the old labels. I think we should keep the old labels and add new ones. In the case of repeat, we keep the new one.

Reply:
you are right, and i was just being lazy! thanks mister senior principal to keep me straight! 👍

Reply:
You troll nicely :smile: